pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -181,11 +213,18 @@ func (s *spdxParser) GetPredicates(ctx context.Context) *assembler.IngestPredica
 
 	preds := &assembler.IngestPredicates{}
 
-	toplevel := s.getPackageElement("DOCUMENT")
+	topLevel := s.getPackageElement("DOCUMENT")
 	// adding top level package edge manually for all depends on package
-	if toplevel != nil {
-		preds.IsDependency = append(preds.IsDependency, common.CreateTopLevelIsDeps(toplevel[0], s.packagePackages, s.filePackages, "top-level package GUAC heuristic connecting to each file/package")...)
-		preds.HasSBOM = append(preds.HasSBOM, common.CreateTopLevelHasSBOM(toplevel[0], s.doc))
+	if topLevel != nil {
+		for _, topLevelPkg := range topLevel {
+			preds.HasSBOM = append(preds.HasSBOM, common.CreateTopLevelHasSBOM(topLevelPkg, s.doc))
```

Comment:
nit : paired with the comment of just needing the spdx ids, we can retrieve the topLevelPkg from packagesPackages

Reply:
note: only need this if we use heuristic, otherwise we can look up in packagePackages